src/Apps/Artwork/Components/Pageviews.tsx

```diff
@@ -0,0 +1,102 @@
+import { color, Sans } from "@artsy/palette"
+import { Pageviews_artwork } from "__generated__/Pageviews_artwork.graphql"
+import { PageviewsQuery } from "__generated__/PageviewsQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PageviewsIcon } from "Styleguide/Elements/icons/Pageviews"
+
+interface Props {
+  artwork: Pageviews_artwork
+}
+
+export class Pageviews extends Component<Props> {
```

Comment:
SFC

Reply:
good call! (this could maybe be a Danger rule even)

Reply:
Its more of a linter rule -- if component doesn't contain state, it should be an SFC. (I'm sorry for always flagging this in PR after PR, but basically everything should be an SFC and the upcoming React version effectively deprecates classes.)